package/yast2-country.spec

```diff
@@ -40,6 +40,7 @@ BuildRequires:  yast2-core >= 3.1.12
 BuildRequires:  yast2-ruby-bindings >= 3.1.26
 # OSRelease.id
 BuildRequires:  yast2 >= 3.2.9
+BuildRequires:  rubygem(%{rb_default_ruby_abi}:ruby-dbus)
```

Comment:
Do we really need that for building the package? The `read_locale_conf` call is mocked in the tests and I'm not sure if DBus is actually running in the build chroot... And using DBus during package build is strange, I'd avoid this dependency if possible.